exercises/affine-cipher/affine-cipher.R

```diff
@@ -0,0 +1,8 @@
+encrypt <- function(message, a, b) {
+
+}
+
+
+decrypt <- function(encryption, a, b) {
```

Comment:
Does `encryption` mean the ciphertext here? If yes, please consider renaming it. `message` -> `plaintext` might also be a good idea then, because those are the [domain terms](https://en.wikipedia.org/wiki/Ciphertext).

Reply:
Yes. Agreed.